roles/cobbler/tasks/main.yml

```diff
@@ -1,4 +1,69 @@
 ---
+- name: Include cobbler keys.
+  include_vars: "{{ secrets_path | mandatory }}/cobbler_keys.yml"
```

Comment:
I haven't seen cobbler_keys.yml yet, but I'm wondering if whatever is there could just be in the group_vars/cobbler.yml in the *-secrets repos.

Is this file encrypted?  I suppose that wouldn't work then.